agents/cryomech-cpa1114/cryomech_cpa1114_agent.py

```diff
@@ -175,13 +184,15 @@ def start_acq(self, session, params=None):
 
             self.take_data = True
             
-            #Publish data, waiting 1/f_sample seconds in between calls. 
+            #Publish data, waiting 1/f_sample seconds in between calls.
             while self.take_data:
                 pub_data = {'timestamp':time.time(), 'block_name':'ptc_status'}
-                data = self.ptc.get_data()
+                data_flag, data = self.ptc.get_data()
                 pub_data['data'] = data
                 time.sleep(1./self.f_sample)
```

Comment:
The sleep is before the publish... it would be better to have it after, right?

Reply:
Yep it's cleaner that way, I made the change